app/controllers/candidate_interface/candidate_interface_controller.rb

```diff
@@ -23,7 +23,7 @@ def current_candidate
     def set_user_context(candidate_id = current_candidate&.id)
       Sentry.set_user(id: "candidate_#{candidate_id}")
 
-      return unless current_candidate
+      return unless current_candidate || resume_session
```

Comment:
```suggestion
      return unless authenticated?
```
There is an `authenticated?` method available through the [Authentication concern](https://github.com/DFE-Digital/apply-for-teacher-training/blob/2ed7dd241a0ac603c7e382b7429f3d4c65c2abc7/app/controllers/concerns/authentication.rb#L6)

I think we should be using that "everywhere"